### DISCUSSION TITLE
Where can I download android-raspberry-pi-ota?

### DISCUSSION THREAD

**Discussion by RetoHB** [tesla-android/issue-tracker#300] [Q&A] [answered]
So far I was not able to find the offline OTA file on Github... Can anyone direct me to the correct location? Thanks

**Comment by mikegapinski** [comment, marked as answer]
https://github.com/tesla-android/android-raspberry-pi-ota/releases/tag/2023.40.1

Od: Reto ***@***.***>
Data: wtorek, 3 października 2023 o 15:18
Do: tesla-android/issue-tracker ***@***.***>
DW: Subscribed ***@***.***>
Temat: [Tesla Android] Where can I download android-raspberry-pi-ota? (Discussion #300)

So far I was not able to find the offline OTA file on Github... Can anyone direct me to the correct location? Thanks

**Reply by RetoHB** [reply]
Just found it now, thanks. ;-) I would just copy the zip file to an USB key and the Android updater will suggest it as an update?
